src/Sylius/Bundle/LocaleBundle/Templating/Helper/LocaleHelper.php

```diff
@@ -37,9 +37,9 @@ public function __construct(LocaleConverterInterface $localeConverter)
     /**
      * {@inheritdoc}
      */
-    public function convertCodeToName($localeCode)
+    public function convertCodeToName($localeCode, $locale = 'en')
     {
-        return $this->localeConverter->convertCodeToName($localeCode);
+        return $this->localeConverter->convertCodeToName($localeCode, $locale);
```

Comment:
Let's make both tests separate?  `it_converts_locales_code_to_name_with_provided_locale` and `it_converts_locales_code_to_name_with_default_locale`